ExposureApp/CRExposureNotification/Core/AppDelegate.swift

```diff
@@ -12,7 +12,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     func application(_ application: UIApplication,
                      didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
-        ExposureManager.shared
+        let _ = ExposureManager.shared
```

Comment:
I was just about to do the same thing :D

Reply:
Nice :D